Tests/Rules/UseConsistentWhitespace.tests.ps1

```diff
@@ -243,10 +255,132 @@ $x = @{
+
+        It "Should find a violation if there is no space after pipe" {
+            $def = 'Get-Item |foo'
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
+            Test-CorrectionExtentFromContent $def $violations 1 '' ' '
+        }
+
+        It "Should find a violation if there is no space before pipe" {
+            $def = 'Get-Item| foo'
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
+            Test-CorrectionExtentFromContent $def $violations 1 '' ' '
+        }
+
+        It "Should not find a violation if there is 1 space before and after a pipe" {
+            $def = 'Get-Item | foo'
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings | Should -Be $null
+        }
+
+        It "Should not find a violation if a backtick is before the pipe" {
+            $def = @'
+Get-Item `
+| foo
 '@
             $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings | Should -Be $null
         }
 
+        It "Should not find a violation if a new-line is after the pipe" {
+            $def = @'
+Get-Item |
+    foo
+'@
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings | Should -Be $null
+        }
 
+        It "Should not find a violation if a backtick is after the pipe" {
+            $def = @'
+Get-Item |`
+foo
+'@
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings | Should -Be $null
+        }
     }
+
+
+    Context "CheckInnerBrace" {
+        BeforeAll {
+            $ruleConfiguration.CheckInnerBrace = $true
+            $ruleConfiguration.CheckOpenBrace = $false
+            $ruleConfiguration.CheckOpenParen = $false
+            $ruleConfiguration.CheckOperator = $false
+            $ruleConfiguration.CheckPipe = $false
+            $ruleConfiguration.CheckSeparator = $false
+        }
+
+        It "Should find a violation if there is no space after opening brace" {
+            $def = 'if ($true) {Get-Item }'
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
+            Test-CorrectionExtentFromContent $def $violations 1 '' ' '
+        }
+
+        It "Should find a violation if there is no space before closing brace" {
+            $def = 'if ($true) { Get-Item}'
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings
+            Test-CorrectionExtentFromContent $def $violations 1 '' ' '
+        }
+
+        It "Should not find a violation if there is 1 space after the opening brace and 1 before the closing brace" {
+            $def = @'
+if($true) { Get-Item }
+'@
+            $violations = Invoke-ScriptAnalyzer -ScriptDefinition $def -Settings $settings | Should -Be $null
+        }
+
+        It "Should not find a violation if there is 1 space inside empty curly braces" {
+            $def = @'
+if($true) { }
```

Comment:
should there be a violation if there is more than 1 space?

Reply:
There will be a violation and it will correct it to have only exactly 1 whitespace between braces. I added a test case for that as well